templates/ingress/ingress.yaml

```diff
@@ -127,8 +172,15 @@ spec:
       paths:
       - path: {{ .notary_path }}
         backend:
+{{- if semverCompare "<1.19-0" .Capabilities.KubeVersion.GitVersion }}
```

Comment:
The `pathType` should be specified when version >=1.19

Reply:
Thanks for the catch! I had tested without notary 🤦  Fixed it